packages/studio-base/src/components/OpenDialog/Start.tsx

```diff
@@ -410,21 +409,17 @@ function SidebarItems(props: { onSelectView: (newValue: OpenDialogViews) => void
 }
 
 export default function Start(props: IStartProps): JSX.Element {
-  const { supportedLocalFileExtensions = [], onSelectView } = props;
+  const { onSelectView } = props;
   const { recentSources, selectRecent } = usePlayerSelection();
   const { classes } = useStyles();
   const analytics = useAnalytics();
 
   const startItems = useMemo(() => {
-    const formatter = new Intl.ListFormat("en-US", { style: "long" });
-    const supportedLocalFiles = formatter.format(
-      Array.from(new Set(supportedLocalFileExtensions)).sort(),
-    );
     return [
       {
         key: "open-local-file",
         text: "Open local file",
-        secondaryText: `Supports ${supportedLocalFiles} files.`,
+        secondaryText: "Visualize directly from your local filesystem.",
```

Comment:
```suggestion
        secondaryText: "Visualize data directly from your local filesystem.",
```

I think having in-app documentation about what the supported file formats are is still useful. Could we combine both?